api/openapi/api.yaml

```diff
@@ -676,5 +676,42 @@ paths:
                     ]
         "500":
           description: Check schematic log. 
+      tags:
+        - Schema Operation
+
+  /explorer/get_node_ranges:
```

Comment:
You are calling `get_node_range` function in `routes.py`? Here it should be `get_node_range`